integrationos-unified/src/unified.rs

```diff
@@ -366,7 +370,7 @@ impl UnifiedDestination {
                     "Mapping request body {}\nUsing js {js}",
                     serde_json::to_string_pretty(&body)
                         .map_err(|e| {
-                            error!("Could not convert body to pretty string {body:?}: {e}");
+                            error!("Failed to convert body to pretty string for connection model. ID: {}, Body: {}, Error: {}", config.id, body, e);
```

Comment:
Security concern: The error logging in here may expose sensitive user data by logging the full request body. We should modify this to log only non-sensitive information.